### PR TITLE
General Functionality

### PR DESCRIPTION
**Pull request by chescobarf**
### Functionality to Monitor App

- Front base components (Monitor.tsx)
- Back base services

### Back Services

- Order by status
- Total orders by status
- Notifications by status
- Set Notification

### CODE REVIEW

File: Admin-Component/messages/pt.json [hunk 1]
```diff
@@ -1,9 +1,6 @@
 {
```
Comment (VioletaLibertad) [inline Admin-Component/messages/pt.json]: Please add a `TODO:` comment so we don't forget to add he portuguese translation 👌

File: Admin-Component/react/components/Monitor.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+type AppProps = {
+   estado: string;
+}
+function Monitor ({estado}:AppProps) {
+  const [toggle,setToggle] = useState(true);
+  const [isLoad, setLoad] = useState(false);
+  const [status, setStatus] =useState("");
+   const[notification,setNotification]=useState()
+
```
Comment (VioletaLibertad) [inline Admin-Component/react/components/Monitor.tsx:14]: Please add a TODO comment to give this functions a functionality

File: Admin-Component/react/components/Monitor.tsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+				<InputButton placeholder="Emails" size="regular" label="Emails to notify" button="Add" type="email" onChange={handleOnChange} />
+			</form>
+			<div className="statusEmails container">
+				{array.map((email,i)=>(	
+					<Tag key={i} className="emailsTag" bgColor="#134CD8" color="#ffffff" onClick={()=>removeEmail(email.name)}>{email.name}</Tag>	
+				))}
+			</div>
+	  		{/* <div className="statusEmails container">
+			  <Tag  className="emailsTag" bgColor="#134CD8" color="#ffffff" onClick={() => console.log('callback')}>christian.escobar@vtex.com.br</Tag>
+			</div> */}
```
Comment (VioletaLibertad) [inline Admin-Component/react/components/Monitor.tsx]: Delete this commented lines

File: Admin-Component/react/components/Monitor.tsx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+         <InputButton placeholder="Emails" size="regular" label="Emails to notify" button="Add" type="email" onChange={handleOnChange} />
+      </form>
+      <div className="statusEmails container">
+         {array.map((email,i)=>(	
+            <Tag key={i} className="emailsTag" bgColor="#134CD8" color="#ffffff" onClick={()=>removeEmail(email.name)}>{email.name}</Tag>	
+         ))}
+      </div>
+        {/* <div className="statusEmails container">
+        <Tag  className="emailsTag" bgColor="#134CD8" color="#ffffff" onClick={() => console.log('callback')}>christian.escobar@vtex.com.br</Tag>
+      </div> */}
```
Comment (VioletaLibertad) [inline Admin-Component/react/components/Monitor.tsx]: Delete this commented lines

File: Admin-Component/react/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vtex.admin-example",
+  "name": "vtexchile.admin-example",
```
Comment (VioletaLibertad) [inline Admin-Component/react/package.json:2]: I think we need to define what vendor should we set in here, and update all packages.json accordingly

File: Admin-Component/react/tsconfig.json [hunk 6]
```diff
@@ -9,8 +9,7 @@
     "sourceMap": true,
     "target": "es2017",
     "skipLibCheck": true,
-    "strict": true
-  },
+    "strict": true  },
```
Comment (VioletaLibertad) [inline Admin-Component/react/tsconfig.json:12]: I think this was unnecessary. What do you think?

File: Service-Component/docs/README.md [hunk 7]
```diff
@@ -125,3 +125,49 @@ We have an (for now, VTEX-only, internal) Splunk dashboard to show all metrics r
 After linking this app and making some requests, you can select `vtex.service-example` and see the metrics for your app. **Don't forget to check the box Development, as you are linking your app in a development workspace**.
 
 For convenience, the link for the current version: https://splunk7.vtex.com/en-US/app/vtex_colossus/node_app_metrics?form.time.earliest=-30m%40m&form.time.latest=%40m&form.picked_context=false&form.picked_region=aws-us-east-*&form.picked_service=vtex.service-example
+
```
Comment (VioletaLibertad) [inline Service-Component/docs/README.md:128]: Before we finish the app, we should update all this to english. Maybe leave a TODO message?

File: Service-Component/node/middlewares/notifications.ts [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import axios from "axios"
+export async function notifications(ctx: Context, next: () => Promise<any>) {
+
+//Status es sacado desde el mismo context y saca el parametro enviado "status" por la URL
+ const  status = ctx.vtex.route.params.status
+// const key="eyJhbGciOiJFUzI1NiIsImtpZCI6IkE3MDZFRDdEMjBDQ0RCMDk0M0ZFMzc0OUVDQTUwQkY1NENDNzFCMTgiLCJ0eXAiOiJqd3QifQ.eyJzdWIiOiJjaHJpc3RpYW4uZXNjb2JhckB2dGV4LmNvbS5iciIsImFjY291bnQiOiJjaHJpc3RpYW5lc2NvYmFyIiwiYXVkaWVuY2UiOiJhZG1pbiIsInNlc3MiOiI1M2RlZjIxNy0yNDFhLTRiN2MtODgzNS04MmQ2YTg4NGI2ZGEiLCJleHAiOjE2MjM0MTg1NDMsInVzZXJJZCI6IjdkMjFjNDg3LWZmMDEtNDE5MS04MGVlLTFjZjY2MDhmMzViNyIsImlhdCI6MTYyMzMzMjE0MywiaXNzIjoidG9rZW4tZW1pdHRlciIsImp0aSI6IjQ0OGI0YWU5LWUwNjktNDY5OC05NmJhLTRhNmI2ZjA3YjkxNiJ9.7m0Tv6e56jK8x6gjKsFFZypBnd0pCmVEhIs7TRh9MXrFtA7tYYRq4VDwH7mMw-W3jd-rNW-KSS8Afq54DHVF_g"
```
Comment (VioletaLibertad) [inline Service-Component/node/middlewares/notifications.ts]: Please delete this comment :D

File: Service-Component/node/middlewares/totalOrders.ts [hunk 9]
```diff
@@ -0,0 +1,31 @@
+import axios from "axios"
+export async function totalOrders(ctx: Context, next: () => Promise<any>) {
+
+//Status es sacado desde el mismo context y saca el parametro enviado "status" por la URL
+const  status = ctx.vtex.route.params.status
+const key = ctx.vtex.adminUserAuthToken
+// const key="eyJhbGciOiJFUzI1NiIsImtpZCI6IkE3MDZFRDdEMjBDQ0RCMDk0M0ZFMzc0OUVDQTUwQkY1NENDNzFCMTgiLCJ0eXAiOiJqd3QifQ.eyJzdWIiOiJjaHJpc3RpYW4uZXNjb2JhckB2dGV4LmNvbS5iciIsImFjY291bnQiOiJjaHJpc3RpYW5lc2NvYmFyIiwiYXVkaWVuY2UiOiJhZG1pbiIsInNlc3MiOiI1M2RlZjIxNy0yNDFhLTRiN2MtODgzNS04MmQ2YTg4NGI2ZGEiLCJleHAiOjE2MjM0MTg1NDMsInVzZXJJZCI6IjdkMjFjNDg3LWZmMDEtNDE5MS04MGVlLTFjZjY2MDhmMzViNyIsImlhdCI6MTYyMzMzMjE0MywiaXNzIjoidG9rZW4tZW1pdHRlciIsImp0aSI6IjQ0OGI0YWU5LWUwNjktNDY5OC05NmJhLTRhNmI2ZjA3YjkxNiJ9.7m0Tv6e56jK8x6gjKsFFZypBnd0pCmVEhIs7TRh9MXrFtA7tYYRq4VDwH7mMw-W3jd-rNW-KSS8Afq54DHVF_g"
```
Comment (VioletaLibertad) [inline Service-Component/node/middlewares/totalOrders.ts]: Delete this line

**Review by VioletaLibertad — CHANGES_REQUESTED** [review]
Great job Chis! 🎉  Just some comments we can review later if you want 👌

File: Admin-Component/react/package.json (hunk 5 above)
Comment (chescobarf) [reply Admin-Component/react/package.json:2]: No aplica

**Review by VioletaLibertad — APPROVED** [review]

**Pull request merged**